hack/deploy

```diff
@@ -14,13 +14,92 @@ readonly clustercheck="$SP/clustercheck"
 readonly curlcheck="$SP/curlcheck"
 readonly swarmcheck="$SP/swarmcheck"
 readonly servicescheck="$SP/servicescheck"
+readonly terraformdir="$SP/../bootstrap/terraform-aws"
+readonly stacks_to_deploy="amp1 amp2 amp3"
+readonly terraform="docker run -v $HOME/.aws:/root/.aws:ro -v $terraformdir:/data --workdir=/data hashicorp/terraform"
+
+# IP range that should have access to the docker remote API
+readonly rapi_cidr="$(curl -sf ifconfig.co)/32"
```

Comment:
For what it's worth, you can make the curl call faster by just querying the ip: `ifconfig.co/ip`.